bin/templates/cordova/lib/pluginHandlers.js

```diff
@@ -34,7 +34,7 @@ var handlers = {
             // a later plugins release.  This is for legacy plugins to work with Cordova.
 
             if (options && options.android_studio === true) {
-                dest = studioPathRemap(obj);
+                dest = studioPathRemap(obj) || dest;
```

Comment:
It is not possible to cherry-pick or rebase this change to work on master. This change will be clobbered by another solution that works more easily on master.